addons/modules/CfgVehicles.hpp

```diff
@@ -0,0 +1,50 @@
+class CfgVehicles {
+    class Logic;
+    class Module_F: Logic {
+        class AttributesBase {
+            class Edit;
+            class Checkbox;
+            class ModuleDescription;
+        };
+        class ModuleDescription {
+            class AnyBrain;
+        };
+    };
+
+    class GVAR(modulePoliceStation): Module_F {
+        scope = 2;
+        displayName = CSTRING(Police_Station);
```

Comment:
If this component will hold more than one module type I think it would make sense to prefix the translation strings with module name:
```sqf
CSTRING(PoliceStation_Name)
CSTRING(PoliceStation_LocationName)
CSTRING(PoliceStation_LocationName_Description)
CSTRING(PoliceStation_HasHelipad)
```

Reply:
Yes, this is good idea. I will add prefixes.